tools/topology/topology1/sof/pipe-eq-iir-waves-codec-playback.m4

```diff
@@ -0,0 +1,122 @@
+# Low Latency Passthrough with EQIIR and Waves codec Pipeline
+#
+# Pipeline Endpoints for connection are :
+#
+#  host PCM_P -- B0 --> EQIIR0 -> B1 --> Waves -- B2 --> sink DAI0
```

Comment:
In fact eq-iir is only needed by dts, not by waves. pipe-waves-codec-playback is enough.

Reply:
@johnylin76 you are right! Add Barry from Waves, @barry-waves suggest that remove the eqiir from this. Because the eqiir only requires when demux to 4ch speakers. Will update the PR soon.